docs/faq.md

```diff
@@ -0,0 +1,61 @@
+cp -r ./* ~/.gazebo/models/.
+```
+Now when the demo is re-launched, Gazebo will be able to find the required models.
+
+
+#### Add/Modify models in the Gazebo World
+The position and orientation of models is defined using the `traffic_editor`. Load the desired `.project.yaml` file into `traffic_editor` to add or modify models into the floor plan.
+
+
+#### Running the simulation with Ignition
+Work on supporting the simulations in Ignition is underway. The setup instructions will be updated in the near future. 
+
+
+#### Does RMF work with robots running navigation stacks? The robots in the demo only travel in straight line segments.
+
+Yes. RMF works independent of the technique used by the robot to localize itself and navigate through coordinates in a map. This is because RMF deals with planning at a higher, traffic level which considers the presence of other robot fleets in the same space. RMF maintains a `schedule database` of the intended paths all robots operating in the same environment and monitors the same for potential conflicts that may arise. The costmap is not an occupancy grid of sensor-detected obstacles but rather a database of spaciotemporal representation of robot trajectories. These trajectories are interpolated as cubic splines and account for the respective robot's kinematics.
+
+A trajectory is submitted to the `schedule database` by an `rmf_fleet_adapter` which interfaces with the proprietary `fleet manager` or robot API. The `rmf_fleet adapters` are configurable depending on the level of control offered by the vendor robot fleet. In general a `rmf_fleet_adapter` keeps track of the trajectories of its robots and computes new plans, when a conflict is detected, by entering into a negotiation with the conflicting robot's `rmf_fleet_adapter`. This is possible as each `rmf_fleet_adapter` maintains a mirror of the `schedule database` and hence is aware all the trajectories active in the space. The generated plan may ask the robot to wait at a node or reroute itself along other lanes in the graph. The result of the plan is communicated to the robot's `fleet manager` via a `PathRequestMsg`. The expectation is that the vendor `fleet manager` will comply to the incoming request and command the respective robot to reach follow the waypoints in the `PathRequestMsg` to reach its destination, conflict free. Here the robot will make use of its navigation stack and drivers to drive to each of the waypoints while avoiding sensor-detected obstacles. In the meanwhile, the `rmf_fleet_adapter` expects the `fleet manager` to update the current position of the robot and its remaining path back to it via a `RobotStateMsg`. Then for example, if a robot pauses due to a dynamic obstacle in its path, the information is implicitly conveyed via the `RobotStateMsg` and this delay is submitted to the `schedule database` which is now visible to the other `rmf_fleet_adapters`, and can inturn replan their robot routes if affected by the delay.
```

Comment:
probably should be similar to `rmf_fleet_msgs` and call them `PathRequest` message and `RobotState` message

Reply:
Thanks for catching that!